.travis/mysql_settings.py

```diff
@@ -44,8 +44,8 @@
         "NAME": "evennia",
         "USER": "evennia",
         "PASSWORD": "password",
-        "HOST": "localhost",
-        "PORT": "",  # use default port
+        "HOST": "127.0.0.1",
+        "PORT": "3306",  # use default port
```

Comment:
Comments would be good. Is there a purpose to these changes?

Reply:
Not part of the docs refactor.